### PR TITLE
Fixed crash: password and user should be escaped with percent charact…

### PR DESCRIPTION
**Pull request by sebastienwindal**
…ers.

### CODE REVIEW

File: Sources/MongoDriver.swift [hunk 1]
```diff
@@ -20,7 +21,9 @@ public class MongoDriver: Fluent.Driver {
         the given database name, credentials, and port.
     */
     public init(database: String, user: String, password: String, host: String, port: Int) throws {
-        let server = try Server("mongodb://\(user):\(password)@\(host):\(port)", automatically: true)
+        let escapedUser = user.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
+        let escapedPassword = password.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
```
Comment (tanner0101) [inline Sources/MongoDriver.swift]: Can we throw an error instead of force unwrapping the password?

**Review by tanner0101 — CHANGES_REQUESTED** [review]

**Review by tanner0101 — APPROVED** [review]

**Comment by tanner0101** [comment]
Is this good to merge?

**Comment by sebastienwindal** [comment]
yes it is!

**Comment by tanner0101** [comment]
Cool. Thanks :)

**Pull request merged**